README.adoc

```diff
@@ -26,27 +26,21 @@ You’ll explore how to check the health of a microservice with MicroProfile Hea
 == What you'll learn
 
 You will learn how to use the MicroProfile Health to report the health of
-a microservice. A health check allows your microservice to participate in the decision-making process about its health.
+a microservice. A health check allows your microservice to participate in the decision-making process about
+its health.
 
-The MicroProfile Health provides an endpoint that includes the health check results as a
-JSON object. This HTTP request endpoint returns a status code of `200` if the application outcome
-is `UP`. Otherwise, the endpoint returns a status code of `503` if the application outcome is `DOWN`.
+MicroProfile Health allows services to report their health and it publishes overall health status to a defined
+endpoint.  A service reports `UP` if it is available and reports `DOWN` if it is unavailable.  MicroProfile Health reports
+individual service status at the endpoint and indicates the overall status as `UP` if all the services are `UP`.  A service
+orchestrator can then use the health statuses to make decisions.
```

Comment:
MicroProfile Health allows services to report their health[,] and it publishes [the] overall health status to a defined endpoint. <--Add a comma and "the."
MicroProfile Health reports [an] individual service status at the endpoint and indicates the overall status as `UP` if all the services are `UP`. <--Add an "an."

Reply:
Fixed